azerothcore/azerothcore-wotlk
# fix(DB/character_skills): Fix Riding Skill messed up by PR #6015

## Changes Proposed:
- Set Riding Skill (762) to 75/75 where Apprentice Riding (33388) is max
- Set Riding Skill (762) to 150/150 where Journeyman Riding (33391) is max
- Set Riding Skill (762) to 225/225 where Expert Riding (34090) is max
- Set Riding Skill (762) to 300/300 where Artisan Riding (34091) is max


## Issues Addressed:
- Closes #6634
- Closes more cases than the one reported 


## SOURCE:
- The issues caused by #6015 were fixed by #6631 but may have left "residue" in `character_skills`
- This PR to fix (at least) affected Riding Skills is more general than PR  #6671


## Tests Performed:
-  SQL statements executed
-  Checked in game


## How to Test the Changes:
- On a character with max Apprentice Riding use `.setskill 762 76 76`
- On a character with max Journeyman Riding use `.setskill 762 151 151`
- On a character with max Expert Riding use `.setskill 762 226 226`
- On a character with max Artisan Riding use `.setskill 762 301 301`
- Shutdown the server
- Run the SQL statements from this PR on the `acore_characters` DB
- Restart the server
- Verify that Riding Skill `value`and `max` have been corrected on the test characters


## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
